APIs_Storage/DataFlashBlockDevice/mbed_app.json

```diff
@@ -0,0 +1,7 @@
+{
+    "target_overrides": {
+        "K64F": {
+            "target.components_add": ["DATAFLASH"]

```

Comment:
Should it be `"*"` as we expect any device that compiles this example to have a Dataflash?